### ISSUE TITLE
Assertion

### ISSUE THREAD

**Issue by gin66** [nicklockwood/Euclid#46]
While performing `text("Hello World!")` the code hits the assertion in `Paths.swift`:
```
internal extension Path {
    init(unchecked points: [PathPoint], plane: Plane?, subpathIndices: [Int]?) {
        assert(sanitizePoints(points) == points) <======
        self.points = points
        self.isClosed = pointsAreClosed(unchecked: points)

```

Triggering character is "!".

BTW: Running on iOS

**Comment by gin66** [comment]
Failed test case is here for [reference](https://github.com/gin66/Euclid/tree/textAssert)

**Comment by nicklockwood** [comment]
@gin66 fixed in 0.4.5
